html/browsers/windows/auxiliary-browsing-contexts/opener-closed.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<meta charset="utf-8">
+<html>
+  <head>
+    <title>Auxiliary Browing Contexts: window.opener when Opener Removed/Closed</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+  </head>
+  <body>
+    <div id="log"></div>
+    <script>
+    setup (() => {
+      localStorage.clear();
```

Comment:
Thank you so much for this feedback and I will update accordingly. I had picked up the pattern I used here from some pre-existing tests and recall thinking, _huh...is this really OK?_ 

Guess it's not the best pattern! Will fix.